### DISCUSSION TITLE
How to migrate vault from normal bitwarden to bitwarden_rs?

### DISCUSSION THREAD

**Discussion by FlyingToto** [dani-garcia/vaultwarden#1288] [Q&A]
Good Afternoon,
I am considering switching from a normal bitwarden install (on docker in Linux) to bitwarden_rs (on docker in Linux). Is there a proper way to migrate the existing vault into the one for bitwarden_rs? can I just reuse the existing folder?

Thanks and regards,

ff

PS: I did check in the wiki but didn't notice any mention of that process, if I missed it please let me know where...

**Comment by polyzen** [comment]
bitwarden_rs requires a web vault build with a few patches: https://github.com/dani-garcia/bw_web_builds

Which distro? Have you looked at https://github.com/dani-garcia/bitwarden_rs/wiki/Third-party-packages ?

**Reply by FlyingToto** [reply]
Hi! 
I was just planning to use the docker image "bitwardenrs/server:latest"

**Reply by polyzen** [reply]
https://github.com/dani-garcia/bitwarden_rs/wiki/Which-container-image-to-use#image-updates:
`The web vault is the only exception; as it's bundled with the bitwarden_rs image, the web vault version is always properly matched to the bitwarden_rs server version. If you only use the web vault as the client (unlikely), then you don't need to worry about these compatibility issues.`

**Reply by FlyingToto** [reply]
ok sorry but now I am a bit more confused... 

- What's the difference between bitwarden_rs and bw_web_builds? 
(It sounds like bitwarden_rs doesn't include the web vault application and that's what bw_web_build is for?)
- My original question was just about how to move my DB content from a "normal" bitwarden docker setup into the db of bitwarden_rs? (can I just copy the content of the sqllite folder?)

**Reply by polyzen** [reply]
Oh sorry, got the terminology mixed up. bw_web_builds has patches and builds of the [web vault](https://github.com/bitwarden/web) that work with bitwarden_rs.

Only thing I found on migrating is https://github.com/dani-garcia/bitwarden_rs/issues/801#issuecomment-570809798

**Reply by FlyingToto** [reply]
kk, so it looks I might need to do an export/import per user.... I will check the cli then.
